packages/components/src/components/segmented-control/segmented-control/segmented-control.tsx

```diff
@@ -47,6 +49,9 @@ export class SegmentedControl {
   /** Sets the initial value of the segmented-control. */
   @Prop() public value?: string | number;
 
+  /** Sets the width and number of columns of the items. */
+  @Prop() public column?: BreakpointCustomizable<number | 'auto'> = 'auto';
```

Comment:
Shouldn't this be called `columns`? Also width is based on columns, so the comment is a bit strange.